src/Message/AbstractRequest.php

```diff
@@ -144,7 +144,8 @@ public function __construct(ClientInterface $httpClient, HttpRequest $httpReques
     }
 
     /**
-     * Initialize request with parameters
+     * Initialize request with parameters.
+     * For new products, the default TAX_CLASSIFICATION is Tax Exempt
```

Comment:
any reason tax classification is all capitalized and with an underscore?

Reply:
I did that because the original variable name is that and I find it usually easier when looking at it outside an IDE doing cmd+F, but no strong preference here.

Reply:
that variable is on Vimeo's private github :-) that should have no influence on this repository--this should be as clear as possible for anyone who may be using it